apps/src/lab2/views/components/aiTutor2/useAiTutor2.tsx

```diff
@@ -12,6 +12,7 @@ import moduleStylesFixed from '../AiTutor2ResponseFixed.module.scss';
 import moduleStylesShrink from '../AiTutor2ResponseShrink.module.scss';
 
 export function useAiTutor2(
+  isEnabled: boolean,
   getFullPrompt: (question: string) => string,
   type: AiTutor2MessageType,
   shrink = false
```

Comment:
optional: might be worth creating an arguments object for this for readability?

Reply:
Good call.  Might do this in a follow-up!